Moasa/Moasa/View/DetailView/DetailView.swift

```diff
@@ -39,15 +39,8 @@ struct DetailView: View {
                         } else { // 기간순 정렬
                             let filtereditems = items.sortbyDate(categoryName: category.category,
                                                                  startDate: startDate, endDate: endDate)
```

Comment:
```suggestion
                            let filteredItems = items.sortbyDate(categoryName: category.category,
                                                                 startDate: startDate, endDate: endDate)
```

변수 네이밍에서, 단어들이 이어질 때에는 camelCase가 가시적으로 보기에 더 좋은 것 같아요
실제로 swift 컨벤션 가이드라인에서도 '변수 네이밍의 경우 camelCase를 따른다' 라고 알려주고 있습니다👍

아래 링크에서 convention 파트를 보시면 될 것 같아요
Ref) https://www.swift.org/documentation/api-design-guidelines/#promote-clear-usage

Reply:
놓친 부분이 있군요 고치고 push 하겠습니다!!